### PR TITLE
Spanish language update v5

### PR DESCRIPTION
**Pull request by Underforest**
Added a lot of missing keys and removed unneeded ones

### CODE REVIEW

File: resources/lang/es/beatmapsets.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 return [
     'availability' => [
         'disabled' => 'Este beatmap no está actualmente disponible para descargar.',
-        'parts-removed' => 'Partes de este beatmap han sido removidas por solicitud de su creador o un titular de derechos de autor.',
+        'parts-removed' => 'Partes de este beatmap han sido elimandas por la solicitud de su creador o un titular de derechos de autor.',
```
Comment (Repflez) [inline resources/lang/es/beatmapsets.php]: Error ortográfico.

**Comment by peppy** [comment]
In the future please name your branch something that isn't `master` before creating a PR.

**Comment by Underforest** [comment]
OK, got that

File: resources/lang/es/community.php [hunk 2]
```diff
@@ -35,6 +35,15 @@
             Me gustaría ofrecer mi agradecimiento personal a aquellos que han apoyado osu! hasta ahora,
             y también a aquellos que continúen apoyando este increíble juego y a la comunidad en el futuro :).',
 
+        'supporter_status' => [
+            'contribution' => '¡Gracias por tu apoyo hasta ahora! ¡Has contribuido un total de :dollars con la compra de :tags tags!',
+            'gifted' => ':giftedTags de tus compras de tags han sido regaladas (con tu toal de :giftedDollars regalados), ¡qué generoso!',
```
[the remark this answers is not in the document]
Comment (Underforest) [reply resources/lang/es/community.php]: Yeah, gonna fix in another pull that

**Pull request merged**